redisvl/index/index.py

```diff
@@ -935,6 +945,16 @@ async def clear(self) -> int:
 
         return total_records_deleted
 
+    async def drop_keys(self, keys: Union[str, List[str]]) -> None:
```

Comment:
Sorry one quick thing. `delete` returns the count of keys dropped. With the `clear` method, I though it might be valuable to return the count here. Should be easy to do the same